src/compiler/scala/tools/nsc/ast/TreeGen.scala

```diff
@@ -174,7 +174,7 @@ abstract class TreeGen extends scala.reflect.internal.TreeGen with TreeDSL {
    */
   def convertToTypeName(tree: Tree): Option[RefTree] = tree match {
     case Select(qual, name) => Some(Select(qual, name.toTypeName))
-    case Ident(name)        => Some(Ident(name.toTypeName).copyAttrs(tree))
+    case Ident(name)        => Some(Ident(name.toTypeName))
```

Comment:
@som-snytt do you have memories about https://github.com/scala/scala/commit/4f40dc87e4 ?

Reply:
March 3 is when you told me about `// javaVersion` so I still owe you one from that week.

Reply:
It seems to have been needed for the "first take" described at https://github.com/scala/bug/issues/5715#issuecomment-1059776386  where (guessing a bit) in
```
new `then`
```
the ident loses the attachment that says it was backquoted. Maybe that only matters post-typer for tree printing now.